tests/director/tests/formation_constraint_api_test.go

```diff
@@ -309,6 +327,15 @@ func TestListFormationConstraintsForFormationTemplate(t *testing.T) {
 	t.Logf("Attaching second constraint to formation template")
 	fixtures.AttachConstraintToFormationTemplate(t, ctx, certSecuredGraphQLClient, constraintSecond.ID, formationTemplate.ID)
 
+	// Assert the two constraints are attached only to the first formation template
+	t.Logf("Get formation template with name %q and id %q, and assert there are is one constraint attached to it", formationTemplate.Name, formationTemplate.ID)
+	formationTemplateOutput = fixtures.QueryFormationTemplateWithConstraints(t, ctx, certSecuredGraphQLClient, formationTemplate.ID)
+	assert.Equal(t, formationTemplateOutput.FormationConstraints, []graphql.FormationConstraint{*constraint, *constraintSecond})
+
+	t.Logf("Get formation template with name %q and id %q, and assert there are no constraints attached to it", secondFormationTemplate.Name, secondFormationTemplate.ID)
+	secondFormationTemplateOutput = fixtures.QueryFormationTemplateWithConstraints(t, ctx, certSecuredGraphQLClient, secondFormationTemplate.ID)
+	assert.Empty(t, secondFormationTemplateOutput.FormationConstraints)
```

Comment:
I think we don't need this, the reason is the same as the comment above?

Reply:
As mentioned above, it can be removed, but I don't think it interferes in any way. Also, we are checking if the sub-resolver is working correctly.